### PR TITLE
Me county fips code

### PR DESCRIPTION
**Pull request by saipraveen18**

### CODE REVIEW

File: components/benefit_markets/app/models/benefit_markets/locations/us_county.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module BenefitMarkets
+  module Locations
+    class UsCounty
```
Comment (saimekala07) [inline components/benefit_markets/app/models/benefit_markets/locations/us_county.rb:5]: This model looks similar to [CountyZip Model ](https://github.com/ideacrew/enroll/blob/trunk/components/benefit_markets/app/models/benefit_markets/locations/county_zip.rb#L7-7 ), what do you think of adding the `fips_code` field CountyZip model

File: app/models/address.rb [hunk 2]
```diff
@@ -111,6 +111,11 @@ def to_a
     [kind, address_1, address_2.to_s, city, state, zip]
   end
 
+  def fetch_county_fips_code
```
Comment (saimekala07) [inline app/models/address.rb:114]: 👍

File: components/benefit_markets/app/models/benefit_markets/locations/us_county.rb (hunk 1 above)
Comment (saipraveen18) [reply components/benefit_markets/app/models/benefit_markets/locations/us_county.rb:5]: We can place counties code over here but currently, in Enroll we are using the countyzip's model to load counties and zips only for a particular state, but for our case we need to load all the Us counties.

File: app/views/events/addresses/_address.xml.haml [hunk 3]
```diff
@@ -7,6 +7,8 @@
     %address_line_3= address.address_3
   %location_city_name= address.city
   %location_state_code= address.state
+  - if EnrollRegistry[:enroll_app].setting(:county_fips_code_enabled).item && address.fetch_county_fips_code.present?
```
Comment (saimekala07) [inline app/views/events/addresses/_address.xml.haml:10]: one last comment, can you add spec for this change
refer here(spec/views/events/v2/addresses/_address.xml.haml_spec.rb) if we don't have the spec for address.xml.haml

Comment (saipraveen18) [reply app/views/events/addresses/_address.xml.haml:9]: Added spec

**Pull request closed without merging**